Doc fixes started 8 6 21 (#841)
nautobot/nautobot: nautobot/docs/models/extras/relationship.md

## Relationship Filters

Filters can be defined to restrict the type or selection of objects for either side of the connection. 

> ### Relationship Filter Example

Reviewer: I don't know that this subheading is really necessary here, as this is the only subsection under the parent section heading.

> As an example, let's create a relationship between Circuits and Devices. 
> In our situation we only would terminate Circuits on Devices with the Role of `edge`.
> 
> To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
> limit the Relationship to only Devices with Role of `edge`:

Reviewer: ```suggestion
As an example, let's create a relationship between Circuits and Devices. 
In our situation we only would terminate Circuits on Devices with the Device Role of `edge`.

To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
limit the Relationship to only Devices with a Device Role whose slug is `"edge"`:
```

> 

Reviewer: Might be helpful somewhere in here to explain how to determine what filters are available for a given model:

- by reading the code (here, `nautobot/dcim/filters.py` -> `DeviceFilterSet` class)
- by checking the filter options available in the REST API (e.g., `http://.../api/docs` and in this case checking the `dcim_devices_list` API endpoint)

Author: love this example because I learned something

> ```

Reviewer: ````suggestion
```json
````

{
    "role": [
        "edge"
    ]
}
```

For context, here is an image of the entire Relationship:

> ![Image of Relationship with json filter](../../img/relationship_w_json_filter.png)

Reviewer: Cropping of this image is a bit odd - lots of whitespace on the left but the right side is cut off?

Author: fixed

Now, the Circuit Relationship field will show up on a Device with an `edge` role:

> ![Image of Edge Device Relationships](../../img/edge_dev_ckt_relationship.png)

Reviewer: `ckt` is not immediately obvious to me. Why not `edge_device_circuit_relationship.png` and `leaf_device_no_circuit_relationship.png` as filenames?

The Circuit Relationship field will *not* show up on a Device with a role `leaf`:

![Image of Leaf Device Relationships](../../img/leaf_dev_no_ckt_relationship.png)

## Relationship Labels

> Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. From the VLANs example above, you might label the relationship so that on the VLANs side the connection appears as 'Devices' and on the Device side the connection appears as 'VLANs'. 

Reviewer: Perhaps:

```suggestion
Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. From the devices/circuits example above, you might label the relationship so that on the Device side the connection appears as 'Terminated Circuits' and on the Circuit side the connection appears as 'Terminating Devices'. 
```
